kube/src/lib.rs

```diff
@@ -130,6 +112,8 @@ cfg_client! {
     pub use api::Api;
     #[doc(inline)]
     pub use client::Client;
+
```

Comment:
Unrelated to this PR, but I found one downside of `cfg_client!`.

`cargo +nightly fmt --all` doesn't work anymore :( Apparently it doesn't look inside macros. `rustfmt +nightly --edition 2018 **/*/*.rs` works.